diff --git a/node_modules/gatsby-transformer-remark/extend-node-type.js b/node_modules/gatsby-transformer-remark/extend-node-type.js
index 90acef8..7efc4a2 100644
--- a/node_modules/gatsby-transformer-remark/extend-node-type.js
+++ b/node_modules/gatsby-transformer-remark/extend-node-type.js
@@ -649,6 +649,19 @@ module.exports = function remarkExtendNodeType({
           }
         }
 
+      },
+      wordCountChinese: {
+        type: "Int",
+        resolve(markdownNode) {
+          return getHTML(markdownNode).then(html => {
+            const pureText = require(`sanitize-html`)(html, {
+              allowTags: []
+            });
+            return (
+              _.words(pureText, /[\s\p{sc=Han}]/gu).length
+            );
+          });
+        }
       }
     });
   });
