diff --git a/node_modules/gatsby-transformer-remark/extend-node-type.js b/node_modules/gatsby-transformer-remark/extend-node-type.js
index 30477b7..01c44a8 100644
--- a/node_modules/gatsby-transformer-remark/extend-node-type.js
+++ b/node_modules/gatsby-transformer-remark/extend-node-type.js
@@ -657,6 +657,19 @@ module.exports = ({
           }
         }
 
+      },
+      wordCountChinese: {
+        type: "Int",
+        resolve(markdownNode) {
+          return getHTML(markdownNode).then(html => {
+            const pureText = require(`sanitize-html`)(html, {
+              allowTags: []
+            });
+            return (
+              _.words(pureText, /[\s\p{sc=Han}]/gu).length
+            );
+          });
+        }
       }
     });
   });
