diff --git a/node_modules/imagemin-pngquant/index.js b/node_modules/imagemin-pngquant/index.js
index c564e97..7cf248b 100644
--- a/node_modules/imagemin-pngquant/index.js
+++ b/node_modules/imagemin-pngquant/index.js
@@ -63,7 +63,7 @@ const imageminPngquant = (options = {}) => input => {
 	const promise = subprocess
 		.then(result => result.stdout) // eslint-disable-line promise/prefer-await-to-then
 		.catch(error => {
-			if (error.code === 99) {
+			if (error.exitCode === 99) {
 				return input;
 			}
 
